app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,30 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{:width => "100%"}
+  %table.column{:align => "left"}
+    %tr
+      %td
+        %h3
+          = t(".customer_greeting", name: @order.bill_address.firstname)
+        %h4
+          = t(".instructions_html", distributor: @order.distributor.name )
+        %img.float-left{:src => "#{@order.distributor.logo.url(:medium)}"}/
```

Comment:
Is the backslash at the end of this line there for a reason?

Reply:
There's no reason. Sorry about that :sweat_smile: